### ISSUE TITLE
No matching DirectoryIndex forbidden

### ISSUE THREAD

**Issue by jpaulino04** [OSEHRA/popHealth#154]
Hello,

I am unable to get the application started.

Apparently the index file can not be found Cannot serve directory /home/pophealth/popHealth/public/: No matching DirectoryIndex.

By the way, I do not see any index anywhere in the application.

Does anyone know how all works?

After install and going to the server ip I get a "403 forbidden" error.

Thanks,

**Comment by petercyli** [comment]
Can you let me know which version of pophealth that you trying to install?

**Comment by jpaulino04** [comment]
@petercyli 

Thank you for replying.
I am trying to install version 5.1.

**Comment by lrasmus** [comment]
@jpaulino04 - apologies if you have checked this, but I just want to confirm you did all of the steps to setup Apache and Passenger from the setup guide?  Also, can you confirm that the path you have listed (/home/pophealth/popHealth/public) exists - I only mention this because we ran into an issue where we had "/home/popHealth" instead of "/home/pophealth".

**Comment by jpaulino04** [comment]
@petercyli 

Yes, the path "/home/pophealth/popHealth/public"  exists but there is no index inside the public folder.

/home/pophealth is my home directory, and "popHealth" is the folder cloned from github.

Also note that I have the all validations passing:

Validating installation...

 * Checking whether this Passenger install is in PATH... ✓
 * Checking whether there are no other Passenger installations... ✓
 * Checking whether Apache is installed... ✓
 * Checking whether the Passenger module is correctly configured in Apache... ✓

Everything looks good. :-)

I am able to start apache, but I still have the forbidden error. I have tried changing the permissions and ownership, but no luck.

**Comment by lrasmus** [comment]
Thanks for confirming all of that @jpaulino04!  I don't believe you will see any index files created - the routing of that is handled magically by Passenger and Rails.  Not sure if any of these considerations would apply: https://www.phusionpassenger.com/library/admin/apache/troubleshooting/ruby/#apache-reports-a-403-forbidden-error

**Comment by jpaulino04** [comment]
@petercyli 
Thank you for that information, I have actually already tried those steps, but Ill give it another try.

Regarding the installation guide,
I have tried following that guide, but some steps don't work and I have had to modify them.

Do you know of a different guide?

**Comment by jpaulino04** [comment]
Hey, I think I got the app working, thank you for your help.

Like I mentioned that guide is outdated.

I had to install passenger using this guide, and basically do the same thing when configuring apache:
https://www.phusionpassenger.com/library/install/apache/install/oss/xenial/

Notice enabling the module, for example should be:
sudo a2enmod passenger
sudo apache2ctl restart

**Comment by lrasmus** [comment]
Sorry to keep asking things you've already tried @jpaulino04, but thanks for the updates and the resolution!  Please feel free to submit a PR for edits to the wiki pages to improve the documentation.

**Comment by jpaulino04** [comment]
@lrasmus 
Thank you for helping out, your comments were actually helpful.
I'll submit that PR when I have the chance.
Thanks!
